src/content/en/2020/markup.md

```diff
@@ -699,10 +698,9 @@ As a rule of thumb and for [usability reasons](https://www.nngroup.com/articles/
 
 <p class="note">Within the latest Safari and Firefox versions, setting <code>target="_blank"</code> on <code>a</code> elements implicitly provides the same <code>rel</code> behavior as setting <code>rel="noopener"</code>. This is already <a href="https://chromium-review.googlesource.com/c/chromium/src/+/1630010">implemented in Chromium</a> as well and will land in Chrome 88.</p>
 
-{# TODO(editors, authors): The following two sections each read like conclusions. Consider merging. #}
-## Status and trends
+## Conclusion
 
-We've sprinkled some observations throughout the chapter, and you'll have made your own observations. At the end of this 2020 analysis, here are some things that stood out for us.
+At the end of this 2020 analysis, here are some things that stood out for us.
```

Comment:
I think this is a little short, from a layout perspective, as we then launch straight into that big figure.

![Conclusion](https://user-images.githubusercontent.com/10931297/99402052-d2d6a600-28e0-11eb-8635-0e485ef13977.png)

I'd prefer a bit more of an introductory paragraph to the conclusion so that sentence doesn't look so lonesome.

I think we want to keep the big figure and our convention is to list big figures first and then talk about them. And even without that it would be a little short IMHO.

Here's what I suggested previously, which is a little longer and retains some of your original intent (even if not your exact wording!), but possibly that's still a little short. Or am I over thinking this?

```suggestion
We've touched on some observations throughout the chapter, but as a reflection on the state of markup in 2020, here are some things that really stood out for us:
```

![image](https://user-images.githubusercontent.com/10931297/99402447-45e01c80-28e1-11eb-9d7e-abc6b7db2c3b.png)

What do you think?

Reply:
LGTM! (Would leave out “really” but that’s a nit, I’m not attached to this.)

Reply:
Removed the `really`.